Issue: Integration tests fail for faiss vector store

### Issue you'd like to raise.

Integration tests for faiss vector store fail when run. 
It appears that the tests are not in sync with the module implementation.

command: poetry run pytest tests/integration_tests/vectorstores/test_faiss.py 

Results summary:

======================================================= short test summary info =======================================================
FAILED tests/integration_tests/vectorstores/test_faiss.py::test_faiss_local_save_load - FileExistsError: [Errno 17] File exists: '/var/folders/nm/q080zph50yz4mcc7_vcvdcy00000gp/T/tmpt6hov952'
FAILED tests/integration_tests/vectorstores/test_faiss.py::test_faiss_similarity_search_with_relevance_scores - TypeError: __init__() got an unexpected keyword argument 'normalize_score_fn'
FAILED tests/integration_tests/vectorstores/test_faiss.py::test_faiss_invalid_normalize_fn - TypeError: __init__() got an unexpected keyword argument 'normalize_score_fn'
FAILED tests/integration_tests/vectorstores/test_faiss.py::test_missing_normalize_score_fn - Failed: DID NOT RAISE <class 'ValueError'>
=============================================== 4 failed, 6 passed, 2 warnings in 0.70s ===============================================

### Suggestion:

Correct tests/integration_tests/vectorstores/test_faiss.py to be in sync with langchain.vectorstores.faiss

--- a/tests/integration_tests/vectorstores/test_faiss.py
+++ b/tests/integration_tests/vectorstores/test_faiss.py
@@ -93,7 +93,7 @@ def test_faiss_with_metadatas_and_filter() -> None:
     )
     assert docsearch.docstore.__dict__ == expected_docstore.__dict__
     output = docsearch.similarity_search("foo", k=1, filter={"page": 1})
-    assert output == []
+    assert output == [Document(page_content="bar", metadata={"page": 1})]
 
 
 def test_faiss_search_not_found() -> None:

